CITATION.cff

```diff
@@ -178,3 +178,16 @@ references:
     notes: >-
       MapReduce techniques are used via Parsl apps and workflow configuration
       to help achieve scalable data engineering for CytoTable.
+  - authors:
+      - name: "Colas Lab"
```

Comment:
The authors should be: Aashna Lamba and Alexandre Colas. Is there a way to credit individuals?

Reply:
Thanks, yes, these can be added!

Reply:
Added in credit to those individuals.